.env.test

```diff
@@ -0,0 +1,8 @@
+MONGODB_URI=mongodb://localhost:27017/test
```

Comment:
This has the advantage of letting you bake the setting of `NODE_ENV` into NPM scripts, rather than having to set them manually.